### PR TITLE
Add Qwen3-Omni (dense) OpenVINO export and inference support

### PR DESCRIPTION
**Pull request by sgonorov**
### What does this PR do?
Adds OpenVINO export and inference support for the Qwen3-Omni dense multimodal model (text + vision + audio).

The model is exported as 6 sub-models: language model, text embeddings, vision patch embeddings, vision merger (with deepstack features), vision position embeddings, and audio encoder. Inference supports text-only, image, audio, and combined image+audio inputs through OVModelForVisualCausalLM.

Key implementation details:

Language model exports hidden_states alongside logits for correct stateful transformation
4D position IDs for Qwen3-Omni's multimodal RoPE (vs 3D in Qwen3-VL)
Audio inference replicates the original model's windowed chunking pipeline before calling the exported audio encoder
Vision merger uses InputEmbeddingPatcher for the position embedding sub-model to work around inspect.signature resolution issues after prior exports
Note: Requires transformers from commit 3d1a4f5e for Qwen3-Omni dense variant support (not yet in a stable release). The Talker subsystem (speech synthesis) is not present in the dense 4B variant and is not supported.

[CVS-174395](https://jira.devtools.intel.com/browse/CVS-174395)

### Before submitting

- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ]  Did you make sure to update the documentation with your changes?
- [x] Did you write any new necessary tests?

### CODE REVIEW

File: optimum/exporters/openvino/model_configs.py [hunk 1]
```diff
@@ -287,6 +287,10 @@ def init_model_configs():
         "transformers",
         "AutoModelForImageTextToText",
     )
+    TasksManager._CUSTOM_CLASSES[("pt", "qwen3_omni", "image-text-to-text")] = (
```
Comment (Wovchena) [inline optimum/exporters/openvino/model_configs.py:354]: I suspect a new omni task is needed

File: tests/openvino/utils_tests.py [hunk 2]
```diff
@@ -172,6 +172,7 @@
     "qwen3": "optimum-intel-internal-testing/tiny-random-qwen3",
     "qwen3_moe": "optimum-intel-internal-testing/tiny-random-qwen3moe",
     "qwen3_vl": "optimum-intel-internal-testing/tiny-random-qwen3-vl",
+    "qwen3_omni": "optimum-intel-internal-testing/tiny-random-qwen3-omni",
```
Comment (Wovchena) [inline tests/openvino/utils_tests.py:321]: Doesn't exist

Comment (sgonorov) [reply tests/openvino/utils_tests.py:321]: Fixed

**Comment by byungilm** [comment]
@sgonorov  I still cannot export int4 weight IR from HF with ce96158. It might have an issue in int4 reload logic for qwen3_omni.
Could it be possible to check to generate int4 weight model?

I tried  the following command :
optimum-cli.exe export openvino --model Qwen3-Omni-4B-Instruct-multilingual --task text-to-audio --weight-format int4 qwen3_omni_4B_ov_int4

**Comment by byungilm** [comment]
Succeeded in generating int4 weight IR with the latest commit `87fce87`.

> @sgonorov I still cannot export int4 weight IR from HF with [ce96158](https://github.com/huggingface/optimum-intel/commit/ce96158521de387c69ba0cbc70a76e3f1e668111). It might have an issue in int4 reload logic for qwen3_omni. Could it be possible to check to generate int4 weight model?
> 
> I tried the following command : optimum-cli.exe export openvino --model Qwen3-Omni-4B-Instruct-multilingual --task text-to-audio --weight-format int4 qwen3_omni_4B_ov_int4
